### DISCUSSION TITLE
LOF fiber specifications

### DISCUSSION THREAD

**Discussion by SaikrishnanR** [Telecominfraproject/oopt-gnpy#490] [Q&A] [answered]
Hello community,

LOF fiber is given as a default fiber type in the eqpt_config, but I couldn't find enough details about what the LOF fiber stood for or its uses and advantages over the other fiber types, other than having a larger effective_area value.  Please shed some light on this topic.

Thank you in advance.

**Comment by gborrach** [comment, marked as answer]
Dear @SaikrishnanR,

As you mentioned, the fiber type name "LOF" (large optical fiber) refers generically to a fiber with a large effective area; in fact, the products on the market may have similar characteristics but different names. As a consequence of the large effective area, the other important characteristic of these fibers is the low attenuation that can be achieved by constructing what is commonly called pure silica core fiber (PSCF). This special type of fiber is particularly used for applications where very low attenuation and non-linearity are required. Below I leave a couple of links where you can find public information on fiber types with characteristics similar to those reported in the equipment configuration file.

https://sumitomoelectric.com/sites/default/files/2020-12/download_documents/80-11.pdf

https://ieeexplore.ieee.org/document/8385829/authors#authors

**Reply by SaikrishnanR** [reply]
Thank you @gborrach for shedding light on this. Will go through the provided links.
